### ISSUE TITLE
undefined symbol: _PyGen_Send

### ISSUE THREAD

**Issue by loynoir** [libfuse/pyfuse3#52]
docker python:3.10
```sh
docker run --mount type=bind,source=$PWD,target=/app,bind-propagation=rshared -w /app --cap-add SYS
_ADMIN --device /dev/fuse -it python:3.10 bash
```
```sh
$ apt update && apt install libfuse3-dev && pip install pyfuse3
$ python
Python 3.10.2
>>> import pyfuse3
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: /usr/local/lib/python3.10/site-packages/pyfuse3.cpython-310-x86_64-linux-gnu.so: undefined symbol: _PyGen_Send
```

docker python:3.9
```sh
docker run --mount type=bind,source=$PWD,target=/app,bind-propagation=rshared -w /app --cap-add SYS
_ADMIN --device /dev/fuse -it python:3.9 bash
```

```sh
$ apt update && apt install libfuse3-dev && pip install pyfuse3
$ python
Python 3.9.10
>>> import pyfuse3
>>>
```

**Comment by astroanax** [comment]
I was on python 10 as well, compiling from source worked for me. I used the [AUR package](https://aur.archlinux.org/packages/python-pyfuse3)

**Comment by Nikratio** [comment]
Thanks for the report! I'm afraid this looks like a bug in the Docker container or the Debian package. If you can reproduce when compiling pyfuse3 from scratch, then please re-open.

**Comment by loynoir** [comment]
@Nikratio 
> If you can reproduce when compiling pyfuse3 from scratch, then please re-open.

### First reproduce & second reproduce: fail
Yes, I'm on archlinux. 

And using docker image python:3.10, 
- to double check something goes wrong
- and easy for you to reproduce

### Before third reproduce
Update system
```sh
$ pacman -Syu
```

### Third: fail
poetry

```bash
$ poetry new reproduce
$ cd reproduce/
$ poetry add pyfuse3
$ poetry shell
$ python
Python 3.10.2
>>> import pyfuse3
ImportError: ~/.cache/pypoetry/virtualenvs/reproduce-{{hash}}-py3.10/lib/python3.10/site-packages/pyfuse3.cpython-310-x86_64-linux-gnu.so: undefined symbol: _PyGen_Send
```

### Fourth: fail
venv

```sh
$ python -m venv venv-reproduce
$ bash
```

```bash
$ source venv-reproduce/bin/activate
$ pip install pyfuse3
$ python
>>> import pyfuse3
ImportError: venv-reproduce/lib/python3.10/site-packages/pyfuse3.cpython-310-x86_64-linux-gnu.so: undefined symbol: _PyGen_Send
```

### Fifth: fail
user global package

```bash
$ pip install pyfuse3
$ python
Python 3.10.2
>>> import pyfuse3
ImportError: ~/.local/lib/python3.10/site-packages/pyfuse3.cpython-310-x86_64-linux-gnu.so: undefined symbol: _PyGen_Send
```

### Sixth: fail
root global package
```bash
$ pip uninstall pyfuse3 -y
$ su
[root] # pip install pyfuse3
[root] # python
Python 3.10.2
>>> import pyfuse3
ImportError: /usr/lib/python3.10/site-packages/pyfuse3.cpython-310-x86_64-linux-gnu.so: undefined symbol: _PyGen_Send
```

### Seventh: OK

AUR python-pyfuse3
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=python-pyfuse3

```sh
[root] # pip uninstall pyfuse3
(exit root)

$ paru -S python-pyfuse3
$ python
Python 3.10.2
>>> import pyfuse3
>>> pyfuse3
<module 'pyfuse3' from '/usr/lib/python3.10/site-packages/pyfuse3.cpython-310-x86_64-linux-gnu.so'
```
### Eighth: fail
Uninstall aur. Reinstall aur. Make sure aur's patched pyfuse3 still work.

`pip uninstall` aur's patched pyfuse3 with root user `pip install`.
 
```sh
$ paru -Rs python-pyfuse3
$ paru -S python-pyfuse3
$ python
Python 3.10.2
>>> import pyfuse3
>>> pyfuse3
<module 'pyfuse3' from '/usr/lib/python3.10/site-packages/pyfuse3.cpython-310-x86_64-linux-gnu.so'

$ sudo pip uninstall pyfuse3 -y
$ sudo pip install pyfuse3
$ python
Python 3.10.2
>>> import pyfuse3
ImportError: /usr/lib/python3.10/site-packages/pyfuse3.cpython-310-x86_64-linux-gnu.so: undefined symbol: _PyGen_Send
```

**Comment by JuniorJPDJ** [comment]
I'm having exactly the same problem when trying to install pyfuse3 in venv.
I installed pyfuse3 from source:
```
pip install --no-binary ":all" --no-cache-dir pyfuse3
```

```
$ python -c 'import pyfuse3'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: /home/juniorjpdj/dev/tgmount/venv/lib/python3.10/site-packages/pyfuse3.cpython-310-x86_64-linux-gnu.so: undefined symbol: _PyGen_Send
```

**Comment by ThomasWaldmann** [comment]
@JuniorJPDJ just guessing: can you recompile the cython stuff using a recent Cython release?

I will try to make a new release of pyfuse3 soon because that will fix some python 3.11 issues (not sure about this issue).
